base-notebook/test/test_container_options.py

```diff
@@ -33,6 +39,13 @@ def test_unsigned_ssl(container, http_client):
     resp = http_client.get("https://localhost:8888", verify=False)
     resp.raise_for_status()
     assert "login_submit" in resp.text
+    logs = c.logs(stdout=True).decode("utf-8")
+    assert "ERROR" not in logs
```

Comment:
That's because sometimes we forget to add good stuff to all the places.
It is perfectly fine to log everywhere in test cases, I will fix this after merging your PR.